docs/Cisco-Asa-Preview-Configuration.md

```diff
@@ -12,7 +12,7 @@
 4. Place the following configuration file on the OMS Agent machine:  
 	* [security_events.conf](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/conf/omsagent.d/security_events.conf)  
 	_Fluentd configuration file to enable collection and parsing of Cisco events_  
-	Destination path on Agent machine: ```/etc/opt/microsoft/omsagent/conf/omsagent.d/```  
+	Destination path on Agent machine: ```/etc/opt/microsoft/omsagent/<workspace id>/conf/omsagent.d/```  
 
   
 5. Configure Cisco ASA event forwarding to the OMS Agent  
```

Comment:
Not for this PR: eventually, once the nxOMSSyslog module gets updated, this will need to be changed to be more flexible with the ports used. But I've made a note to change this when I update the DSC module.